extension-manifest-v2/app/panel-android/actions/blockingActions.js

```diff
@@ -171,7 +171,7 @@ export function trustRestrictBlockSiteTracker({ actionData, state }) {
 	const updated_settings_category = updated_settings_categories.find(category => category.id === cat_id);
 
 	const selectedSettingsTracker = updated_settings_category.trackers
-		.find(tracker => tracker.shouldShow && +tracker.id === app_id);
```

Comment:
This `+` could be significant. It forces a conversion to a Number.

Just to be sure: you are aware of it and tracker.id and app_id have the same type?

Reply:
this is exactly the bug introduced in 8.10.0. Now, tracker id should always be a string. Also because for new trackers we will likely use their names instead of numbers.